src/mlpack/tests/data/t1_test.arff

```diff
@@ -0,0 +1,89 @@
+% analcatdata    A collection of data sets used in the book "Analyzing Categorical Data,"
+%                by Jeffrey S. Simonoff, Springer-Verlag, New York, 2003. The submission
+%                consists of a zip file containing two versions of each of 84 data sets, 
+%                plus this README file. Each data set is given in comma-delimited ASCII
+%                (.csv) form, and Microsoft Excel (.xls) form.
+% 
+% NOTICE: These data sets may be used freely for scientific, educational and/or
+%         noncommercial purposes, provided suitable acknowledgment is given (by citing
+%         the above-named reference).
+% 
+% Further details concerning the book, including information on statistical software
+% (including sample S-PLUS/R and SAS code), are available at the web site
+% 
+%             http://www.stern.nyu.edu/~jsimonof/AnalCatData
```

Comment:
Wow, shortening "Analyzing Categorical Data" to "AnalCatData" is pretty awful!  Thank you for not naming this file `analcatdata.arff` or something... :)  However, should we rename it `braziltourism_test.arff` or something?  I'm not sure what the `t1` refers to; let me know if I'm missing something.

Reply:
Sure, I'll do the renaming. t1 was just a random name.